src/editor.c

```diff
@@ -2394,7 +2394,7 @@ typedef struct
 } SelectionRange;
 
 
-#define CURSOR_PLACEHOLDER "_" /* Would rather use … but not all docs are unicode */
+#define CURSOR_PLACEHOLDER "..." /* Would rather use … but not all docs are unicode */
```

Comment:
S/A https://github.com/geany/geany/commit/957b49b868214a4aea81641dedeabb98a497e4c1#comments

Reply:
Why not use Unicode ellipsis? All documents are UTF-8 in memory and since placeholders markers shouldn't ever be saved (they aren't actual contents of the document), it seems like it would work fine.

Reply:
I had that before. Since nothing prevents you from saving a doc containing the placeholders, I didn't want to enforce utf8 for saved files.

Reply:
Shouldn't the placeholders not be saved since they aren't meant to be literally inserted into the document? IIRC other editors I've tried that do this (namely XCode and QtCreator) don't treat placeholders as text in the document, but rather just as a graphical cue for where you'll bounce when tabbing to the next placeholder in the edit.

Reply:
They are, in fact, inserted literally into the buffer. This is required for Scintilla markers. Since we don't currently filter the document upon save, they are also saved to the file.

What do other editors insert when you save a file containing the placeholder?

Reply:
I guess yeah ideally the placeholder wouldn't be any actual character but simply a display thing, but I'm not sure Scintilla allows for this, does it?  It would need be able to have an indicator that takes more room than the range it's on, which doesn't sounds likely, but I might miss an existing feature.

Otherwise for what the placeholder is I don't care that much; I just didn't feel like `_` looked straightforward to me when using the thing, but I don't have a string opinion either.  I still feel like https://github.com/geany/geany/commit/957b49b868214a4aea81641dedeabb98a497e4c1#commitcomment-23346900, but if the consensus is to keep `_` I won't argue.

Reply:
> I guess yeah ideally the placeholder wouldn't be any actual character but simply a display thing, but I'm not sure Scintilla allows for this, does it?

Since the placeholders are marked with a (presumably) unique indicator, there's no reason they couldn't be removed from the saved contents (mere matter of programming).

Reply:
> What do other editors insert when you save a file containing the placeholder?

IIRC XCode and QtCreator enter some kind of special placeholder-filling mode where you can tab between them and fill them out, and then you commit them.

Reply:
Since we don't agree let's post-pone this very change. The rest of the PR is good and should be merged.